driver.md

```diff
@@ -72,9 +72,49 @@ To do so, we'll extend sort `JSON` with some EVM specific syntax, and provide a
     rule <k> startTx => #finalizeBlock ... </k>
          <txPending> .List </txPending>
 
-    rule <k> startTx => loadTx(#sender(#dasmTxPrefix(TYPE), TN, TP, TG, TT, TV, #unparseByteStack(DATA), CID, TA, TW, TR, TS)) ... </k>
-         <chainID> CID </chainID>
+    rule <k> startTx => loadTx( #TxSender(TXID) ) ... </k>
          <txPending> ListItem(TXID:Int) ... </txPending>
+
+    syntax AcctExp ::= Account
+    syntax KResult ::= Account
+    syntax AcctExp ::= #TxSender( Int )
+ // -----------------------------------
+    rule <k> #TxSender( TXID ) => #sender( LegacyTxData(TN, TP, TG, TT, TV, DATA), TW, TR, TS ) ... </k>
+         <message>
+           <msgID>      TXID </msgID>
```

Comment:
I've added this `#TxSender` cooling rule to `loadTx` to use the new `TxData` symbols.

It's here in `driver.md`, right next to the only place it's needed, but Firefly would benefit from it and it doesn't use `driver.md`. I guess it can go somewhere in `evm.md` or `state-loader.md`, but I'm not sure which.

Reply:
I think because we are just using `strict` to access state, but not to evaluate some expression language (which would require nested unfolding of the expressions using the heating/cooling rules), we can probably use contextual functions here instead:

```
rule [[ #TxSender(TXID) => #sender(...) ]]
       <message>
       ...
       </message>   
```